tests/utilities/test_fetching.py

```diff
@@ -84,3 +91,149 @@ def test_misconfiguration_error():
 
     iter(fetcher)
     assert fetcher.loader_iters
+
+
+def get_cycles_per_ms() -> float:
+    """
+    Measure and return approximate number of cycles per millisecond for torch.cuda._sleep
+    Copied from: github.com/pytorch/pytorch/blob/master/test/test_cuda.py
```

Comment:
Maybe i'm blind but I don't see this function in the link, can you fix it?

Reply:
https://github.com/pytorch/pytorch/blob/master/test/test_cuda.py#L81